### DISCUSSION TITLE
社区版无法录制http请求吗？

### DISCUSSION THREAD

**Discussion by knight-zang** [arextest/arex-agent-java#560] [Q&A]
java -javaagent:./arex-agent.jar -Darex.tags.env=$ENV -Darex.service.name=<service.name> -Darex.storage.service.host=<storage.service.host> -Darex.enable.debug=true -jar <service.name>.jar --spring.profiles.active=$ENV  
我运行上面的命令，虽然录制下来了 但是协议类型都是doubbe 这是为什么呀

**Comment by lucas-myx** [comment]
你所说的协议类型都是doubbe，是笔误还是是指rpc框架Dubbo呢？
如果是dubbo rpc框架的话，说明你们的应用就是dubbo服务，dubbo和其他http封装的框架是不同的类型，请先确认你们使用了哪种http框架，参考arex支持列表：[Http Client](https://github.com/arextest/arex-agent-java?tab=readme-ov-file#http-client)，确认是否在支持范围内
